sql/mysql/updates/1.4.0.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE `#__virtuemart_zasilkovna_branches`;
```

Comment:
Zvážil bych u obou dotazů IF (NOT) EXIST.

Reply:
To mi přijde defenzivní. V jakém případě tam ta tabulka není, když se migruje na 1.4.0 ?

Reply:
ok